src/app/features/fss-search/fss-search.component.ts

```diff
@@ -297,10 +299,23 @@ export class FssSearchComponent implements OnInit {
         this.displayLoader = false;
         this.analyticsService.login();
       })
+      this.fileShareApiService.getBatchAttributes().subscribe((batchAttributeResult) => {
```

Comment:
Can the code in the new subscription for the `getBatchAttributes` event be performed in the existing event handler  on line 82?

Reply:
Forget that. After debugging I see I have misunderstood how `subscribe()` is working and the callback on line 82 is not called after the log-in pop up